app/javascript/mastodon/components/display_name.tsx

```diff
@@ -8,10 +8,11 @@ import { autoPlayGif } from '../initial_state';
 import Skeleton from './skeleton';
 
 interface Props {
-  account: Account;
-  others: List<Account>;
-  localDomain: string;
+  account?: Account;
+  others?: List<Account>;
+  localDomain?: string;
```

Comment:
@renchap I believe these should be optional, as to allow rendering `display_name` as a skeleton. I've also lifted up the logic that determines the `account`, as to avoid the optional prop from causing typescript issues